setup.py

```diff
@@ -19,7 +19,7 @@
         "gcsfs",
         "pandas",
         "pandas-gbq",
-        "pybigquery",
+        "sqlalchemy-bigquery",
```

Comment:
does this also need to go in requirements.txt?

Reply:
yes good catch, I looked for it there but missed it apparently

I just substituted `pybigquery==0.7.0` for `sqlalchemy-bigquer==1.4.3` in both `requirements-dev.txt` and `requirements.txt`, do you that's correct / all that's needed?

Reply:
makes sense to me